src/status_im/utils/ethereum/erc20.cljs

```diff
@@ -140,7 +151,7 @@
 
 (defn get-token-transfer-logs
   ;; NOTE(goranjovic): here we cannot use web3 since events don't work with infura
```

Comment:
This comment is a little weird now that `web3` is used.

Reply:
clarified the comment